java/pom.xml

```diff
@@ -736,6 +738,7 @@
            See https://github.com/jbosstools/m2e-apt/issues/62 for details
       -->
       <activation>
+        <jdk>1.8</jdk>
```

Comment:
Hmm, we already have a 1.8-specific profile below. What's the difference?

Reply:
Changed